contrib/net/http/orchestrion.client.yml

```diff
@@ -94,18 +94,18 @@ aspects:
       # Wire the context that is found to the handlers...
       - wrap-expression:
           imports:
-            client: github.com/DataDog/dd-trace-go/contrib/net/http/v2/client
```

Comment:
Maybe add a comment explaining the "weird" name?

Reply:
done!